sdk/cosmos/azure_data_cosmos/tests/cosmos_containers.rs

```diff
@@ -14,226 +13,223 @@ use azure_data_cosmos::{
 };
 use futures::TryStreamExt;
 
-use framework::{test_data, TestAccount};
-
-#[recorded::test]
-pub async fn container_crud(context: TestContext) -> Result<(), Box<dyn Error>> {
-    let account = TestAccount::from_env(context, None).await?;
-    let cosmos_client = account.connect_with_key(None)?;
-    let db_client = test_data::create_database(&account, &cosmos_client).await?;
-
-    // Create the container
-    let properties = ContainerProperties {
-        id: "TheContainer".into(),
-        partition_key: "/id".into(),
-        indexing_policy: Some(IndexingPolicy {
-            included_paths: vec!["/*".into()],
-            excluded_paths: vec![r#"/"_etag"/?"#.into()],
-            automatic: true,
-            indexing_mode: Some(IndexingMode::Consistent),
-            ..Default::default()
-        }),
-        ..Default::default()
-    };
-
-    let throughput = ThroughputProperties::manual(400);
-
-    let created_properties = db_client
-        .create_container(
-            properties.clone(),
-            Some(CreateContainerOptions {
-                throughput: Some(throughput),
+use framework::{test_data, TestClient};
+
+#[tokio::test]
+pub async fn container_crud_simple() -> Result<(), Box<dyn Error>> {
+    TestClient::run_with_db(|_, db_client| {
+        Box::pin(async move {
```

Comment:
The bodies of these tests should be largely unmodified, something that becomes way more obvious if you enable "Hide Whitespace" in the PR review settings at the top right:

<img width="586" height="594" alt="Image" src="https://github.com/user-attachments/assets/dac6cbed-5352-4cc5-82b0-8b7c9b99c85c" />